libs/community/langchain_community/vectorstores/qdrant.py

```diff
@@ -1367,6 +1367,28 @@ def from_texts(
         qdrant.add_texts(texts, metadatas, ids, batch_size)
         return qdrant
 
+    @classmethod
+    def from_existing_index(
+        cls: Type[Qdrant],
+        embedding: Embeddings,
+        path: str,
```

Comment:
Why is `path` the only explicit param while the others are `**kwargs`?
I'd recommend specifying all the params supported by `_generate_clients()` explicitly for better typing.